extensions/wikia/TemplateClassification/TemplateClassification.i18n.php

```diff
@@ -39,6 +39,7 @@
 	'template-classification-edit-modal-error' => 'Unfortunately we were not able to save this template type. Would you mind trying again?',
 	'template-classification-indicator' => 'Template type:',
 	'template-classification-notification-error-retry' => 'Something went wrong and we could not save the template type. Could you retry it using the link under the page title?',
+	'template-classification-open-modal-key-tip' => 'Open classification dialog (Ctrl+Shift+K)',
```

Comment:
This combination can will vary on different OS. I'm wondering shouldn't we choose just one letter as `accesskey` and then key combination is handle by OS and browser (http://www.w3schools.com/tags/att_global_accesskey.asp)